wiki/Glossary/en.md

```diff
@@ -793,11 +776,11 @@ The amount of slider ticks used in a slider is dependent on the slider velocity,
 
 ### Spectate
 
-Spectate, or "Spec", is when someone watches another player playing a beatmap in real time (with approx. 500ms delay).
+Spectate, or spec, is when someone watches another player playing a beatmap in real time (with approx. 500ms delay).
 To spectate, open the extended chat and click on a grey user panel.
 
 For this to work, you'll need to have a copy of the exact same beatmap the player you wish to spectate is playing.
-However, if you have an osu!supporter tag, the beatmapset will be downloaded for you.
+However, if you have an [![osu!supporter](/wiki/shared/osu!supporter.png)](/wiki/osu!supporter/) tag, the beatmapset will be downloaded and loaded for you.
```

Comment:
It's been the way I've been linking to osu!supporter (using a linked inline image). If this should not be used, I'll note not to do this in the wiki standards in case if I forget.